hs_csdms/utils.py

```diff
@@ -0,0 +1,31 @@
+def split_csdms_names():
+    """ Split each record in csdms file and assign them to corresponding return part
+        :return names, names in csdms
+        :return splitted_names, split each name by space
+        :return decors, decorations for each name in csdms
+        :return splitted_decors, split each decor by space
+
+    """
+    filename = 'hs_csdms/csdms'
```

Comment:
I'm curious how this file was created.  Perhaps if I knew where this file comes from I could make more sense of why/how it is being parsed.

Reply:
This file is from https://csdms.colorado.edu/wiki/CSN_Searchable_List-Names.
We do not have a way to access this list as we did for odm2 (access the file as a web link). I currently copy this list from this website and put it in this file.